app/prompt.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 const (
-	AutoConfirmFlagName = "auto_confirm"
+	AutoConfirmFlagName = "auto-confirm"
```

Comment:
I think should keep this as `auto_confirm` since folks are using it now. That or we could print a helpful message asking them to use auto-confirm instead - but that could break some use-cases folks have.

Reply:
added an alias to the flag, to support both.